.github/workflows/scripts/check_for_changed_pages.js

```diff
@@ -0,0 +1,63 @@
+module.exports = {
+  getChangedPages: ({ github, context, buildDir }) => {
+    const fs = require('fs');
+    const cheerio = require('cheerio');
+    
+    const urlList = [];
+    
+    const {
+      issue: { number: issue_number },
+      repo: { owner, repo }
+    } = context;
+
+    // Use the Github API to query for the list of files from the PR
+    return github
+      .paginate(
+        'GET /repos/{owner}/{repo}/pulls/{pull_number}/files',
+        { owner, repo, pull_number: issue_number },
+        (response) => response.data.filter((file) => (file.status === 'modified' || file.status === 'added'))
+      )
+      .then((files) => {
+        const possiblePages = [];
+        const platforms = [
+          'android',
+          'angular',
+          'flutter',
+          'javascript',
+          'nextjs',
+          'react',
+          'react-native',
+          'swift',
+          'vue',
+        ]
```

Comment:
nit: should we add a semicolon? (just thinking out loud but maybe in the future we can get some linting and stuff for this folder too)

```suggestion
        ];
```

Reply:
Yeah, I'll add a task for that. Updated this in latest!